paper-api/src/main/java/org/bukkit/World.java

```diff
@@ -2805,11 +2803,12 @@ default double getHumidity(int x, int z) {
     /**
      * Gets if beds work in this world.
      *
-     * A non-working bed will blow up when trying to sleep. {@link #isNatural()}
-     * defines if a bed can be used to set spawn point.
+     * A non-working bed can blow up when trying to sleep, but that may
+     * not always be the case.
      *
      * @return true if beds work in this world
      */
+    @Deprecated(since = "1.21.11")
```

Comment:
Need to document the deprecation reason (applies for all the other deprecated members)